microsoft-edge/web-platform/web-platform.md

```diff
@@ -90,3 +91,15 @@ See [Detect Windows 11 using User-Agent Client Hints](how-to-detect-win11.md).
 The `password` input control in Microsoft Edge includes a **password reveal** button.  To make sure that the password is entered correctly, a user can click the **password reveal** button or press `Alt`+`F8`, to show the characters in the password field.  You can remove the **password reveal** control, or customize the control styling.
 
 See [Customize the password reveal button](password-reveal.md).
+
+
+<!-- ====================================================================== -->
+## View formatted JSON responses or files
+
+Sometimes a web server responds to HTTP requests by returning data encoded as JSON.  JSON data can be difficult to read when it's formatted as a single long, concatenated line of text. The same can occur when opening a JSON file from disk.
+
+Microsoft Edge includes the **JSON viewer**, a tool that reformats the JSON data shown in a browser to make it easier to read.
```

Comment:
```suggestion
Microsoft Edge includes the **JSON viewer**, a tool that automatically reformats and highlights the JSON data shown in a browser to make it easier to read.  The **JSON viewer** reformats JSON server responses and local files, and indicates any lines that have syntax errors.

Formatting and highlighting JSON data is useful because sometimes a web server responds to HTTP requests by returning data encoded as JSON.  JSON data can be difficult to read when it's formatted as a single long, concatenated line of text. The same can occur when opening a JSON file from disk.
```